src/witan/models/load_data.clj

```diff
@@ -104,7 +109,7 @@
    Output is map with keyword and core.matrix dataset"
   [keyname filepath]
   (->> (apply-rec-coercion {:type keyname} (load-csv filepath))
-       (dataset-after-coercion)
+       dataset-after-coercion
```

Comment:
The previous step `load-csv`returns a map of column names and data, like:
```Clojure
{:column-names vector-of-headers
 :columns vector-of-data-rows}
```
This step `dataset-after-coercion` actually builds the dataset.